bootstrap/api/v1beta2/ck8sconfig_types.go

```diff
@@ -66,6 +66,20 @@ type CK8sConfigSpec struct {
 	// +optional
 	SnapstoreProxyID string `json:"snapstoreProxyId,omitempty"`
 
+	// Channel is the channel to use for the snap install.
+	// +optional
+	Channel string `json:"channel,omitempty"`
+
+	// Revision is the revision to use for the snap install.
+	// If Channel is set, this will be ignored.
```

Comment:
Not sure if what I'm about to say is the best way to go, but I would prefer to have a validation that checks you've only entered one of these. Otherwise throws and error or warns you that hey things are conflicting. WDYT?